pages/abi-encoder.tsx

```diff
@@ -25,34 +24,72 @@ const Home: NextPage = () => {
         <title>ABI Coder - ERC725 Tools</title>
       </Head>
       <div className="container">
+        <h2 className="title is-2">ABI Encoder</h2>
         <article className="message is-info">
           <div className="message-body">
-            This tool lets you encode and decode{' '}
-            <a href="https://docs.lukso.tech/standards/smart-contracts/lsp0-erc725-account">
-              LSP0 ERC725 Account
-            </a>{' '}
-            transactions.
+            This tool will encode and decode transaction data of
+            <a
+              className="mx-1"
```

Comment:
Not sure why you need this but I'm on mobile so I can't test locally

Reply:
Its either `mx-1` on every link or `{' '}` behind AND in front of the element. Class looks much nicer and failsafe imo. Also saw it in the examples of Bulma.